### PR TITLE
Increase maxRecvMsgSize in grpc server of mako-stub

### PR DESCRIPTION
**Pull request by slinkydeveloper**
This should mitigate the #694

/assign grantr
/area test-and-release

### CODE REVIEW

File: test/mako/stub-sidecar/main.go [hunk 1]
```diff
@@ -32,7 +32,8 @@ import (
 )
 
 const (
-	port = ":9813"
+	port                               = ":9813"
+	defaultServerMaxReceiveMessageSize = 1024 * 1024 * 100
```
Comment (vagababov) [inline test/mako/stub-sidecar/main.go]: can you leave some comment on how is this number inferred?

**Comment by adrcunha** [comment]
/approve

I'll defer LGTM to Victor.

**Comment by dprotaso** [comment]
Out of curiosity can the mako results be streamed? (ie. use multiple messages)

**Comment by slinkydeveloper** [comment]
@dprotaso not sure, this depends on mako client i think

**Pull request merged**